scripts/api-inference/scripts/generate.ts

```diff
@@ -377,13 +377,18 @@ await Promise.all(
 
 // Fetch recommended models
 TASKS.forEach((task) => {
-  DATA.models[task] = TASKS_DATA[task].models;
+  DATA.models[task] = TASKS_DATA[task].models.filter(
+    (model: { inference: string }) =>
+      ["cold", "loading", "warm"].includes(model.inference),
+  );
 });
 
 // Fetch snippets
 // TODO: render snippets only if they are available
 TASKS.forEach((task) => {
-  const mainModel = TASKS_DATA[task].models[0].id;
+  // Let's take as example the first available model that is recommended.
+  // Otherwise, fallback to the first model of the task (better to have a snippet than nothing).
+  const mainModel = DATA.models[task][0]?.id || TASKS_DATA[task].models[0].id;
```

Comment:
SGTM, or otherwise just put `repo_id`, to avoid putting models that wont' work

Reply:
Good idea to use a placeholder instead.
Done in https://github.com/huggingface/hub-docs/pull/1396/commits/67759de6bf1423c4ec491f01702ec9188d3d2e26